presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoRpmControlUtils.java

```diff
@@ -172,7 +175,7 @@ public static Response statusUsingRpm()
         }
         catch (ProcessingException e) {
             LOGGER.info("Presto is not running");
-            return Response.status(OK).entity(GSON.toJson("Presto is installed but not running"))
+            return Response.status(OK).entity(GSON.toJson(format("Presto version '%s' is installed but not running", prestoVersion)))
```

Comment:
This is not a useful JSON response. I would expect a JSON response to look something like this:
```json
{
    "running": true,
    "version": "1.43.6",
    "environment": "Tundra",
    "coordinator": false
}
```
You should be able to achieve this by passing a `Map<String, Object>` or something similar to `Gson::toJson`. Don't worry about getting the values' types right; we can fix those when this is switched to Jackson.